src/site/content/en/fast/codelab-cloudinary-url-api/index.md

```diff
@@ -0,0 +1,334 @@
+{% endAside %}
+
+## ✔︎ Check-in
+
+Your `index.html` file should now look like this:
+
+```html
+<!--
+Copyright 2018 Google LLC.
+SPDX-License-Identifier: Apache-2.0
+-->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Cloudinary URL API Demo: Optimized Images</title>
+    <meta name="Description" content="In this codelab, learn how to use Cloudinary with its simple URL API to optimize images for faster downloads and serve media in next-gen formats. All of that happens dynamically in the Cloud without any build step.">
+    <link href="https://cloudinary-res.cloudinary.com/image/asset/favicon-192x192-d6a96e11dd5adfebbb717d154665ee80.png" rel="icon" sizes="192x192" type="image/png">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>body {text-align: center}img {border: 1px solid black}.wrapper {margin-bottom: 24px}.wrapper .price {font-size: 24px}</style>
+  </head>
+  <body>
+    <header>
+      <h1>
+        Foobar Flowershop
+      </h1>
+      <p>We sell the best flowers!</p>
+    </header>
+    <div class="wrapper">
+      <img src="https://res.cloudinary.com/demo/image/fetch/q_auto,f_auto/https://cloudinary-url-api.glitch.me/images/flower1.png" alt="Yellow bouquet" />
+      <div class="price">Yellow bouquet - $9</div>
+    </div>
+    <div class="wrapper">
+      <img src="https://res.cloudinary.com/demo/image/fetch/q_auto,f_auto/https://cloudinary-url-api.glitch.me/images/flower2.jpg" alt="Cream bouquet" />
+      <div class="price">Cream bouquet - $5</div>
+    </div>
+    <div class="wrapper">
+      <img src="https://res.cloudinary.com/demo/image/fetch/q_auto,f_auto/https://cloudinary-url-api.glitch.me/images/flower3.png" alt="Light pink" />
+      <div class="price">Light pink bouquet - $6</div>
+    </div>
+    <div class="wrapper">
+      <img src="https://res.cloudinary.com/demo/image/fetch/q_auto,f_auto/https://cloudinary-url-api.glitch.me/images/flower4.jpg" alt="Violet bouquet" />
+      <div class="price">Violet bouquet - $8</div>
+    </div>
+    <div class="wrapper">
+      <img src="https://res.cloudinary.com/demo/image/fetch/q_auto,f_auto/https://cloudinary-url-api.glitch.me/images/flower5.jpg" alt="Polka dot" />
+      <div class="price">Polka dot bouquet - $11</div>
+    </div>
+    <div class="wrapper">
+      <img src="https://res.cloudinary.com/demo/image/fetch/q_auto,f_auto/https://cloudinary-url-api.glitch.me/images/flower6.jpg" alt="Burgandy bouquet" />
+      <div class="price">Burgandy bouquet - $7</div>
+    </div>
+  </body>
+</html>
+```
+
+## Image Performance Gains
+
+There are impressive web performance gains by using <a href="https://cloudinary.com/documentation/image_optimization">cloud to optimize images</a> without losing quality.
+
+<div class="w-table-wrapper">
```

Comment:
Glad you liked it. Fun comparison. 

P.S. I use [Numi](https://numi.io/) mac calculator to calculate the optimized image sizes.

P.P.S In case you're into this, I've written a custom extension for Numi calculator where I only have to write `optim(newSize;oldSize)` of an image to calculate the percentage savings.

![optim](https://on.ahmda.ws/29b9a9/c)

Here's the code ↓

```js
/**
 * Optimization percentage in new vs old size.
 *
 * @usage optim(newSize;oldSize)
 * @usage optim(10;100)
 * @param Number newSize The new reduced size.
 * @param Number oldSize The old large size.
 * @author Ahmad Awais <https://twitter.com/MrAhmadAwais>
 */

numi.addFunction({ id: 'optim', phrases: 'optim' }, function(values) {
	return {
		double: (values[0].double / values[1].double) * 100 - 100
	};
});
```